src/Microsoft.AspNetCore.Mvc.RazorPages/UrlHelperExtensions.cs

```diff
@@ -0,0 +1,103 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.AspNetCore.Routing;
+
+namespace Microsoft.AspNetCore.Mvc
+{
+    /// <summary>
+    /// Razor Page specific extensions for <see cref="IUrlHelper"/>.
+    /// </summary>
+    public static class UrlHelperExtensions
```

Comment:
This should have a different name, so that it doesn't overlap with the class in Mvc.Core